src/cljs/main/broadfcui/page/groups/groups_management.cljs

```diff
@@ -0,0 +1,132 @@
+(ns broadfcui.page.groups.groups-management
+  (:require
+    [dmohs.react :as react]
+    [broadfcui.common :as common]
+    [broadfcui.common.components :as comps]
+    [broadfcui.common.flex-utils :as flex]
+    [broadfcui.common.management-utils :refer [MembershipManagementPage]]
+    [broadfcui.common.modal :as modal]
+    [broadfcui.common.style :as style]
+    [broadfcui.common.table.table :refer [Table]]
+    [broadfcui.common.table.style :as table-style]
+    [broadfcui.endpoints :as endpoints]
+    [broadfcui.nav :as nav]
+    [broadfcui.page.groups.create-group :refer [CreateGroupDialog]]
+    [broadfcui.page.workspace.monitor.common :as moncommon]
+    [broadfcui.utils :as utils]
+    ))
+
+
+(react/defc GroupTable
+  {:reload
+   (fn [{:keys [this]}]
+     (react/call :load-data this))
+   :render
+   (fn [{:keys [state this]}]
+     (cond
+       (:error-message @state) (style/create-server-error-message (:error-message @state))
+       (nil? (:groups @state)) [comps/Spinner {:text "Loading groups..."}]
+       :else
+       [Table
+        {:data (:groups @state)
+         :body {:behavior {:reorderable-columns? false}
+                :style table-style/table-light
+                :columns
+                [{:header "Group Name" :initial-width 500 :sort-initial :asc
+                  :sort-by :text
+                  :as-text
+                  (fn [{:keys [group-name]}]
+                    group-name)
+                  :render
+                  (fn [{:keys [group-name access-levels]}]
+                    (if
+                     (contains? (set access-levels) "Owner")
+                      (style/create-link {:text group-name
+                                          :href (nav/get-link :group group-name)})
+                      group-name))}
+                 {:header "Access Levels" :initial-width :auto
+                  :column-data #(clojure.string/join ", " (:access-levels %))}]}
```

Comment:
For functions like this I've been preferring `(comp (partial clojure.string/join ", ") :access-levels)`, but it's not much better.  I just like avoiding a `#` macro when possible, and also avoiding naming a function argument (see: Point-Free Programming).